sycl/plugins/unified_runtime/ur/adapters/level_zero/platform.cpp

```diff
@@ -9,6 +9,32 @@
 #include "platform.hpp"
 #include "ur_level_zero.hpp"
 
+////////////////////////////////////////////////////
+/// L0-Adapter-specific environment variables
+
+/// UR_L0_DEBUG
+/// Enables extra debug capabilities in the L0 adapter.
+/// Mask accepting following values:
+/// UR_L0_DEBUG_NONE = 0x0, default
+/// UR_L0_DEBUG_BASIC = 0x1, prints UR and ZE calls made
+/// UR_L0_DEBUG_VALIDATION = 0x2, enables validation
+///   layer in L0 loader.
+/// UR_L0_DEBUG_CALL_COUNT = 0x4, enables memory leak detection
+int UrL0Debug;
+static const int UrL0DebugReadEnvVar = [] {
```

Comment:
Do we have any other static variables initializations in other TUs that we want to trace? If so, the order of initialization is unspecified and we might be missing some debug events.

Reply:
@aelovikov-intel : I dont think that applies here. UR L0 Debug is to trace inside the L0 backend (adapter/plugin), and urInit below is the first thing being called when using the L0 backend. So mechanism here works with how we are using these env vars inside L0.

@smaslov-intel : what do you think?

Reply:
Yeah, I think we are safe here as we don't want to trace other static initializations.